web/modules/custom/joinup_communities/semic/semic.module

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @file
+ * The SEMIC module.
+ */
+
+use Drupal\Core\Render\Markup;
+
+const SEMIC_COLLECTION_ID = 'http://data.europa.eu/w21/bf8ff383-be51-4d25-ba57-f4fbb194be08';
```

Comment:
Just for testing purposes, it would be nice if we could swap this:

```php
define('SEMIC_COLLECTION_ID', Settings::get('semic.collection_id', 'http://data.europa.eu/w21/bf8ff383-be51-4d25-ba57-f4fbb194be08'));
```